instances/azp-cleanup-snapshots/ami_filter.js

```diff
@@ -0,0 +1,24 @@
+// Functions that assist when filtering through AMIs.
+
+/**
+ * Determines whether the AMI and its snapshots should be cleaned by this
+ * Lambda based on the tags it has.
+ * @param tags - an array of all the tags attached to the AMI.
+ * @returns {boolean} - true if the AMI is in scope and should be cleaned up.
+ * False otherwise. Note that the Lambda only cleans up the AMI if unused.
+**/
+exports.isAmiInScopeFromTags = function (tags) {
```

Comment:
any reason why this doesnt use arrow fun ?

Reply:
Changed to arrow funcs. Thank you.